api/src/routers/meals.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express';
```

Comment:
Am i seeing double :eyeglasses: or are there two meals route files :) I guess you just need to keep one of them.